src/Equinor.ProCoSys.IPO.Infrastructure/Repositories/InvitationRepository.cs

```diff
@@ -31,32 +31,39 @@ public void UpdateProjectOnInvitations(string projectName, string description)
             //Intentionally left blank for now
         }
 
-        public void UpdateCommPkgOnInvitations(string projectName, string commPkgNo, string description)
+        public bool ShouldMoveCommPkg(Guid toProjectGuid, Guid commPkgGuid)
         {
-            var project = _context.Projects.SingleOrDefault(x => x.Name.Equals(projectName));
+            var toProject = _context.Projects.SingleOrDefault(x => x.Guid.Equals(toProjectGuid));
+            var commPkgsToUpdate = _context.CommPkgs.Where(cp => cp.Guid.Equals(commPkgGuid)).ToList();
+            return (commPkgsToUpdate.Where(x => x.ProjectId == toProject.Id)?.Count() > 0);
+        }
 
-            var commPkgsToUpdate = _context.CommPkgs.Where(cp => project != null && cp.ProjectId == project.Id && cp.CommPkgNo == commPkgNo).ToList();
+        public void UpdateCommPkgOnInvitations(Guid commPkgGuid, string description)
+        {
+            var commPkgsToUpdate = _context.CommPkgs.Where(cp => cp.Guid.Equals(commPkgGuid)).ToList();
 
             commPkgsToUpdate.ForEach(cp => cp.Description = description);
         }
 
-        public void MoveCommPkg(string fromProjectName, string toProjectName, string commPkgNo, string description)
+        public void MoveCommPkg(Guid toProjectGuid, Guid commPkgGuid, string description)
         {
-            var toProject = _context.Projects.SingleOrDefault(x => x.Name.Equals(toProjectName));
-            var fromProject = _context.Projects.SingleOrDefault(x => x.Name.Equals(fromProjectName));
+            var toProject = _context.Projects.SingleOrDefault(x => x.Guid.Equals(toProjectGuid));
 
-            var commPkgsToMove = _context.CommPkgs.Where(cp => fromProject != null && cp.ProjectId == fromProject.Id && cp.CommPkgNo == commPkgNo).ToList();
 
-            var mcPkgsToMove = _context.McPkgs.Where(mc => fromProject != null && mc.ProjectId == fromProject.Id && mc.CommPkgNo == commPkgNo).ToList();
+            var commPkgsToMove = _context.CommPkgs.Where(cp => cp.Guid == commPkgGuid).ToList();
+            var fromProjectIds = commPkgsToMove.Select(c => c.ProjectId).ToList();
+            var fromProject = _context.Projects.SingleOrDefault(x => fromProjectIds.Contains(x.Id));
```

Comment:
i think we need a comment or better naming/method use to clarify a bit here. 
There can be more than one commpkg with commPkgGuid?
and we expect all of them to have the same project ID?